### ISSUE TITLE
Temperature tower non-manifold in Linux

### ISSUE THREAD

**Issue by FireflyJar** [OrcaSlicer/OrcaSlicer#6628]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.1.1

### Operating System (OS)

Linux

### OS Version

Linux Mint 21.3 Cin

### Additional system information

Temperature tower non-manifold in Linux and Fix Model not available in Linux.

![image](https://github.com/user-attachments/assets/9ca201fc-9366-40f7-aa4c-8cf037b5c6fa)

### Printer

any

### How to reproduce

Calibration > Temperuture

### Actual results

see screenshot

### Expected results

usable test stl

### Project file & Debug log uploads

[3mf.zip](https://github.com/user-attachments/files/16829145/3mf.zip)

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by FireflyJar** [comment]
looks like it was fixed at one time
https://github.com/SoftFever/OrcaSlicer/issues/343
but still happening in Linux

**Comment by Ergonomicmike** [comment]
https://github.com/SoftFever/OrcaSlicer/issues/3349

**Comment by 67RS427** [comment]
I'm seeing this bug as well on the latest v2.2.0 on MacOS

**Comment by Lud00** [comment]
Same issue on Windows on 2.1.0 and 2.2.0
![image](https://github.com/user-attachments/assets/c5dce62d-80c0-46a8-b697-d04ed4c6087b)

**Comment by melbiz447** [comment]
Same issue here, but cannot fix the model on Mac. Had it on the older version too. Can we resolve this issue? Orca is way better than my primary slicer, but with a new filament...
![temp-tower-orca-2 2 0-mac os 14 7 1](https://github.com/user-attachments/assets/a0a28269-037c-45d0-8867-bd7c22dca273)

**Comment by 67RS427** [comment]
Same issue here on MacOS with Orca v2.2... Please fix, thanks!

**Comment by wisess** [comment]
temperature_tower.stl same issue on the last version

**Comment by yanfali** [comment]
I'm also seeing this on 2.2.0 linux flatpak version

**Comment by rali717** [comment]
Issue on  appImage version Ocrca Slicer 2.3.0 dev, too.

**Comment by PaoloC68** [comment]
Since it is a pretty long time that this bug is around and since it appears that will not be fixed quickly I will recommend anyone that has this problem to go to this website https://www.formware.co/onlinestlrepair and fix the object. That is a far better temporary solution than have to install windows just to fix a 3d object.

**Comment by hho** [comment]
Happens on 2.3.0 on Windows as well (though of course on Windows there actually _is_ a repair function)

**Comment by hho** [comment]
Problem still exists. (Do I now have to write this every 90 days? Stupid bot.)

**Comment by ElonSatoshi** [comment]
<img width="628" height="628" alt="Image" src="https://github.com/user-attachments/assets/6f65f0eb-2ec4-4087-94b3-5b963e5647ba" />

What is it going to do, remove the issue if it's ignored long enough? Lmao

**Comment by LJKraus** [comment]
Non-manifold calibration models have been present since before December 2023, Orca version 1.9.  Unchanged in the current nightly build of 2.3.2-dev.

See Issue #3349 

It is an annoyance, not an actual problem. The models slice and print without repair.

**Comment by RF47** [comment]
This issue was solved in #11861
